internal/pkg/cli/svc_delete.go

```diff
@@ -105,8 +107,10 @@ func newDeleteSvcOpts(vars deleteSvcVars) (*deleteSvcOpts, error) {
 
 // Validate returns an error if the user inputs are invalid.
 func (o *deleteSvcOpts) Validate() error {
-	if o.appName == "" {
-		return errNoAppInWorkspace
+	if o.name != "" || o.envName != "" {
+		if o.appName == "" {
+			return fmt.Errorf("--%s must be provided", appFlag)
+		}
```

Comment:
hmm why do we need this validation? let's say we have `--env prod` then can we prompt users to put in which application they want to use? If `prod` is not in the app they put in, it'll just error out automatically then i guess.

Reply:
Would that prompting happen within Ask()? (It would have to be separate from `AskAppName`, which is using the selector.)

Reply:
i guess if let's say the user doesn't specify `--app` flag then we'll ask them which application they'd like to use, which seems fine to me... that's why i think this validation might not be required.

Reply:
Thanks! Took out that validation. We'll fix the opaque SSM error message in a separate PR.